src/app/data/oasis-account-names.ts

```diff
@@ -65,19 +65,20 @@ const getOasisAccountsMetadata = async (network: Network, layer: Layer): Promise
 export const useOasisAccountsMetadata = (
   network: Network,
   layer: Layer,
-  queryOptions: { enabled: boolean },
+  queryOptions: { enabled: boolean; useErrorBoundary?: boolean },
 ) => {
   return useQuery(['oasisAccounts', network, layer], () => getOasisAccountsMetadata(network, layer), {
     enabled: queryOptions.enabled,
     staleTime: Infinity,
+    useErrorBoundary: queryOptions.useErrorBoundary,
   })
 }
 
 export const useOasisAccountMetadata = (
   network: Network,
   layer: Layer,
   address: string,
-  queryOptions: { enabled: boolean },
+  queryOptions: { enabled: boolean; useErrorBoundary?: boolean },
```

Comment:
Can we somehow use underlying `QueryObserverOptions` interface? Looks painful, if we want to add any additional fields to `queryOptions`, so why not just expose everything?